<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KLFKatePluginConfigWidget</class>
 <widget class="QWidget" name="KLFKatePluginConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,0,0,0,0" columnstretch="1,0,0,0">
   <item row="5" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Latex Preamble</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="KLFLatexEdit" name="txtPreamble"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>path to KLatexFormula executable</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="KUrlRequester" name="pathKLF">
        <property name="mode">
         <set>KFile::ExistingOnly|KFile::File|KFile::LocalOnly</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QCheckBox" name="chkAutoPopup">
     <property name="text">
      <string>Pop-Up Automatically</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Popup Transparency:</string>
     </property>
     <property name="buddy">
      <cstring>sldTransparency</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSlider" name="sldTransparency">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>20</number>
     </property>
     <property name="tracking">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="KIntSpinBox" name="spnTransparency">
     <property name="suffix">
      <string notr="true">%</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>20</number>
     </property>
     <property name="base">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="lblTitle">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>KLatexFormula KTextEditor Plugin (version %1)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Popup Max Size:</string>
     </property>
     <property name="buddy">
      <cstring>sldMaxSize</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSlider" name="sldMaxSize">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>8</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
     <property name="tracking">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="lblMaxSize">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>600x200</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="margin">
      <number>4</number>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextBrowserInteraction</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QCheckBox" name="chkPopupLinks">
     <property name="text">
      <string>Show command links in popup</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KLFLatexEdit</class>
   <extends>QTextEdit</extends>
   <header>klflatexedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>chkAutoPopup</tabstop>
  <tabstop>sldTransparency</tabstop>
  <tabstop>spnTransparency</tabstop>
  <tabstop>sldMaxSize</tabstop>
  <tabstop>pathKLF</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>sldTransparency</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spnTransparency</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spnTransparency</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sldTransparency</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
